vuecomponents/tool/src/main/frontend/src/components/dynamic-criterion.vue

```diff
@@ -2,7 +2,7 @@
   <div class="crit-background">
     <div class="row align-items-center">
       <div class="col-md-2">
-        <BButton @click="toggleColor" :variant="paintSelection" :class="['select-btn']" :disabled="saving || editing" :title="i18n.select_criterion">{{idpos+1}}</BButton>
+        <BButton @click="toggleColor" :variant="paintSelection" :class="['select-btn']" :disabled="saving || editing || criterion.id == null || criterion.id < 1" :title="i18n.select_criterion">{{idpos+1}}</BButton>
```

Comment:
Do you think `!criterion.id` would do, what you are trying to achieve? It would cover `undefined`, `null` and `0`. Or do you need to check negative numbers?

Reply:
Negative numbers shouldn't happen I guess